includes/class-coblocks-block-assets.php

```diff
@@ -655,6 +655,15 @@ protected function is_page_gutenberg() {
 		}
 		return false;
 	}
+
+	/**
+	 * Determine if the page content contains an element with a coblocks-animate class.
+	 *
+	 * @return boolean True when an element on the page has .coblocks-animate class, else false.
+	 */
+	public function has_coblocks_animation() {
+		return false !== strpos( get_the_content(), 'coblocks-animate' );
+	}
```

Comment:
Do you think we should use similar logic here for the content as your Timber fix? I guess it would only affect those on the custom page types we have discussed before.

Reply:
Good call. Done in the latest commit.